### PR TITLE
[8.0][website_mail_snippet_vertical_resize_base] Resize any element

### PR DESCRIPTION
**Pull request by yajo**
This commit extract the vertical resizing plugin part from the
`website_mail_snippet_responsive` module into its own module, to allow
reusing it easily.

It will be needed in the upcoming non-responsive snippets PR.

@rafaelbn @dgzurita

### CODE REVIEW

**Comment by rafaelbn** [comment]
OK, tested in local :+1: thanks!

**Comment by dgzurita** [comment]
Tested in runbot :+1:

File: website_mail_snippet_responsive/__openerp__.py [hunk 1]
```diff
@@ -15,10 +15,9 @@
         "images/snippets.png",
     ],
     "depends": [
-        "website_mail",
+        "website_mail_snippet_vertical_resize_base",
```
Comment (pedrobaeza) [inline website_mail_snippet_responsive/__openerp__.py:18]: Change module version also

**Comment by pedrobaeza** [comment]
The icon for base_resize... can be the same resizer as the one used in other places (something like https://icons8.com/2014/09/30/arrows-resize-horizontal-split-horizontal-split-vertical/)

**Comment by yajo** [comment]
@pedrobaeza Thanks, I used an icon from FontAwesome since I'm not sure about the license of the one you linked. It follows the same idea. :wink:

File: website_mail_snippet_responsive/__openerp__.py (hunk 1 above)
Comment (yajo) [inline website_mail_snippet_responsive/__openerp__.py:18]: Fixed.

**Comment by pedrobaeza** [comment]
Only one thing more: Please make the icon the same size for both height and weight (putting a transparent canvas), or you will get a deformed view in module list.

**Comment by yajo** [comment]
Oops, fixed.

**Comment by rafaelbn** [comment]
"This branch has conflicts that must be resolved" please fix, thanks!

**Comment by yajo** [comment]
Fixed.

File: website_mail_snippet_vertical_resize_base/static/description/icon.svg [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+     id="layer1">
+    <text
+       id="text3340"
+       y="1037.1029"
+       x="49.95929"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:97.27411652px;line-height:87.99999952%;font-family:FontAwesome;-inkscape-font-specification:FontAwesome;text-align:center;writing-mode:lr-tb;text-anchor:middle;opacity:1;fill:#333333;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:1;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1"
+       xml:space="preserve"><tspan
+         y="1037.1029"
+         x="49.95929"
+         id="tspan3342"></tspan></text>
```
Comment (pedrobaeza) [inline website_mail_snippet_vertical_resize_base/static/description/icon.svg:38]: Maybe it's better to convert the text to object for not depending of having the font installed.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**